lib/temping.rb

```diff
@@ -21,7 +21,7 @@ def teardown(clear_dependencies: false)
           end
         end
         @model_klasses.clear
-        ActiveSupport::Dependencies.clear if clear_dependencies
+        ActiveSupport::Dependencies::Reference.clear!
```

Comment:
No. I think that you should have just reverted my changes instead of introducing this check.